Update manage-the-distributed-cache-service.md (#956)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointServer/administration/manage-the-distributed-cache-service.md

## Fine-tune the Distributed Cache service by using a PowerShell script
<a name="finetune"> </a>

**Monitoring**

You can monitor performance counters on the Distributed Cache servers to get a better understanding of cache performance issues.
Some of the [counters](https://docs.microsoft.com/previous-versions/appfabric/ff637725(v=azure.10)) that are typically useful to troubleshoot issues include:

1. %cpu used up by cache service

2. %time spent in GC by cache service

> 3. Total cache misses/sec - A high value here can indicate your application performance might suffer because it is not able to fetch data from cache. Possible causes for this include eviction and/or expiry

Reviewer: ```suggestion
3. Total cache misses/sec - a high value here can indicate your application performance might suffer because it is not able to fetch data from cache. Possible causes for this include eviction, and/or expiry, of items from cache.
```

> of items from cache.

Reviewer: ```suggestion
```

4. Total object count - Gives an idea of how many items are in the cache. A big drop in object count could mean eviction or expiry is taking place.

> 5. Total client reqs/sec - This counter is useful in giving an idea of how much load is being generated on the cache servers from the application. A low value here usually means some sort of a bottleneck

Reviewer: ```suggestion
5. Total client reqs/sec - This counter is useful in giving an idea of how much load is being generated on the cache servers from the application. A low value here usually means some sort of a bottleneck outside of the cache server (perhaps in the application or network), hence very little load is being placed on cache servers.
```

> outside of the cache server (perhaps in the application or network) and hence very little load is being placed on cache servers.

Reviewer: ```suggestion
```

> 6. Total Evicted Objects - If cache servers are constantly evicting items to make room for newer objects in cache, it is usually a good indication that you will need more memory on the cache servers to hold

Reviewer: ```suggestion
6. Total Evicted Objects - If cache servers are constantly evicting items to make room for newer objects in cache, this is usually a good indication that you will need more memory on the cache servers to hold the dataset for your application.
```

> the dataset for your application.

Reviewer: ```suggestion
```
